config/routes.rb

```diff
@@ -33,10 +40,7 @@
   # Scope to change urls to swedish
   scope path_names: {new: 'ny', edit: 'redigera'} do
     namespace :admin do
-      resources :users, path: :anvandare, only: [:index] do
```

Comment:
Don't know why it's displaying that I changed that. But it seems OK now in current commit.